starfish/core/spots/DecodeSpots/decoding_uitls.py

```diff
@@ -0,0 +1,23 @@
+from itertools import product
+
+
+from starfish.core.intensity_table.intensity_table import IntensityTable
+from starfish.core.types import Axes, Features, SpotFindingResults
+
+
+def build_spot_traces_exact_match(spot_results: SpotFindingResults):
+    # create IntensityTable with same x/y/z info accross all r/ch
+    spot_attributes = spot_results.get_spots_for_round_ch({Axes.ROUND: 0, Axes.CH: 0})
```

Comment:
no this line just grabs the spot attributes from the first tile (since there all in the same location) then fills in the intensity for the rest of the rounds/chs and those locations.